README.md

```diff
@@ -46,7 +46,7 @@ downloadEmailAttachments({
   log: console.log,
   since: '2015-01-12',
   lastSyncIds: ['234', '234', '5345'] // ids already dowloaded and ignored, helpful because since is only supporting dates without time
-  attachmentHandler: function (attachmentData, callback) {
+  attachmentHandler: function (attachmentData, callback, errorCB) {
```

Comment:
errrorCB can be called multiple times and aggregates all errors in one execution cycle. the callback may only be used once and marks the end of one function call. this design made debugging a lot easier. and is in this case an acceptable diversion from standard callbacks IMHOP